fixtures/test-utils/advanced-types/index.ts

```diff
@@ -8,6 +8,10 @@ export class TestUtilWrapper {
     return [];
   }
 
+  findSomething(): TestUtilWrapper | null {
```

Comment:
Should we also test it with inferred return type?

Reply:
It does not matter. It is still the same `callSignature.getReturnType()`